src/main/java/me/drepic/proton/managers/RabbitManager.java

```diff
@@ -0,0 +1,147 @@
+package me.drepic.proton.managers;
+
+import com.rabbitmq.client.*;
+import me.drepic.proton.ProtonManager;
+import me.drepic.proton.message.MessageAttributes;
+import me.drepic.proton.message.MessageContext;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+import java.util.concurrent.TimeoutException;
+
+public class RabbitManager extends ProtonManager {
+
+    private Connection connection;
+    private Channel channel;
+    private String queueName;
+
+    private final String host;
+    private final String virtualHost;
+    private final int port;
+    private final String username;
+    private final String password;
+
+    public RabbitManager(String name, String[] groups, String host, String virtualHost, int port, String username, String password) throws Exception {
+        super(name, groups);
+        this.host = host;
+        this.virtualHost = virtualHost;
+        this.port = port;
+        this.username = username;
+        this.password = password;
+        this.connect();
+    }
+
+    public void connect() throws IOException, TimeoutException {
```

Comment:
Add `@Override` for all overridden methods. Also why are these public when the parent class is protected?